prov/rxm/src/rxm.h

```diff
@@ -403,8 +407,16 @@ struct rxm_ep {
 
 struct rxm_conn {
 	struct fid_ep *msg_ep;
+
+	/* Identifier to be posted into rxm_ep::conn_deferred_list.
+	 * This should notify CQ handdling fucntion to progress the
+	 * rxm_conn::deferred_op_list. When the deferred_op_list is
+	 * empty, this connection have to be removed from
+	 * rxm_ep::conn_deferred_list */
```

Comment:
nits: handdling->handling, fucntion->function, 'have to be'->'has to be'

Reply:
Done